cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -0,0 +1,174 @@
+            <div class="form-group">
+
+                {% if input.label %}
+                <label class="form-label-header"
+                       {{ 'for=filter_' ~ input.id if input.id else '' }}>
+                    {{ input.label }}
+                </label>
+                {% endif %}
+
+                {% if input.type == 'form' %}
+                    <div class="form-l">
+                        {{ loop(input.children) }}
+                    </div>
+                {% elif input.type == 'email'
+                   or input.type == 'number'
+                   or input.type == 'password'
+                   or input.type == 'tel'
+                   or input.type == 'text'
+                   or input.type == 'url' %}
+
+                    <input type="{{ input.type }}"
+                           value="{{ input.attrs.value if input.attrs.value else '' }}"
+                           placeholder="{{ input.attrs.placeholder if input.attrs.placeholder else '' }}"
+                           {{ 'id=filter_' ~ input.id if input.id else '' }}
+                           {{ 'name=filter_' ~ input.name if input.name else '' }}>
+                {% elif input.type == 'checkbox' %}
+                    {% for checkbox in input.attrs -%}
+                        <label class="form-group_item"
+                               {{ 'for=filter_' ~ checkbox.id if checkbox.id else '' }}>
+                            {{ _filter_checkbox(checkbox.label,
+                                                checkbox.id,
+                                                checkbox.name,
+                                                checkbox.value) | safe }}
+                        </label>
+                    {% endfor %}
+                {% elif input.type == 'select' %}
+                    <label class="form-group_item">
+                        <select {{ 'id=filter_' ~ input.id if input.id else '' }}
+                                {{ 'name=filter_' ~ input.name if input.name else '' }}>
+                        {% for option in input.attrs -%}
+                            {{ _filter_option(option.label, option.value) | safe }}
+                        {% endfor %}
+                        </select>
+                    </label>
+                {% endif %}
+            </div>
+        </div>
+    {% endfor %}
+{%- endmacro %}
+
+{# A helper macro for filterable_list_controls.render()
+   ========================================================================== #}
+{% macro _filter_checkbox(label, id, name, value) %}
+    <input class="custom-input"
+           type="checkbox"
+           value="{{ value }}"
+           id="filter_{{ id }}"
+           name="filter_{{ name }}"
+           {{ 'id=filter_' ~ id if id else '' }}
+           {{ 'name=filter_' ~ name if name else '' }}
+           {{ 'checked' if is_filter_selected(name, value) else '' }}>
+        {{ label if label else value }}
```

Comment:
One extra indentation here since this isn't wrapped by the input. Also, how does the `label` get the `id`?